indra/assemblers/pybel/assembler.py

```diff
@@ -337,11 +337,14 @@ def _assemble_conversion(self, stmt):
                 node = _get_agent_grounding(agent)
                 # TODO check for missing grounding?
                 pybel_list.append(node)
+        try:
+            rxn_node_data = reaction(
+                reactants=pybel_lists[0],
+                products=pybel_lists[1],
+            )
+        except:
+            return
```

Comment:
I'm not sure in what situation an error can happen here, it might make sense to figure this out and handle it explicitly rather than have a generic try/except here. @ddomingof do you happen to remember why this had to be added?

Reply:
Yes, there errors with some weird nodes that were not grounded and were part of a reaction. There were not many but it was required to catch the error. We can remove this line as this is not necessarily something that should be handled here

Reply:
Unfortunately, I cant push to your branch but yes, I would just remove this,

Reply:
I'm pretty sure I figured out what the corner case was and handled it directly and removed the try/except.